apps/web/lib/slots.ts

```diff
@@ -15,26 +15,29 @@ export type GetSlots = {
   frequency: number;
   workingHours: WorkingHours[];
   minimumBookingNotice: number;
+  eventLength: number;
 };
 export type WorkingHoursTimeFrame = { startTime: number; endTime: number };
 
 const splitAvailableTime = (
   startTimeMinutes: number,
   endTimeMinutes: number,
-  frequency: number
+  frequency: number,
+  eventLength: number
```

Comment:
frequency and eventLength are probably the same thing? I remember frequency was the event time duration in minutes

Reply:
No, frequency is the same as eventLength when the user isn't making use of custom interval for slots. They're different when the user makes use of `Time Slot Intervals`; frequency being the interval chosen while eventLength being the length of the event